docs/subset15.md

```diff
@@ -20,6 +20,7 @@ For programs that are in the subset, the compiler should conform to the Ballerin
    * `any` type
    * `byte` type
    * `anydata` type
+   * `function` type
```

Comment:
Bullet below with "function types" with two sub bullets (one for bare function type)

Reply:
Fixed